main/ertp/guide/amounts.md

```diff
@@ -109,14 +109,14 @@ return a `brand`.
 
 Values are the "how many" part of an `amount`. There are no `value`
 methods, but two `amountMath` methods use or return them. 
-- <router-link to="../api/amount-math.html#amountmath-getvalue-amount">`amountMath.getValue(amount)`</router-link>
+- [`amountMath.getValue(amount)`](../api/amount-math.md#amountmath-getvalue-amount)
```

Comment:
In several places (e.g. line 76 above) I see changes from relative path (../) to absolute (/ertp/) paths but here and below, they're left as relative. Is the change to absolute paths important or not? They seem to produce identical behavior in the generated pages.

Reply:
Some of these were simply wrong, and it was easier to get them right when the path is absolute.